doc/authoring_command_modules/README.md

```diff
@@ -207,8 +207,8 @@ History notes are auto-generated based on PR titles and descriptions starting fr
     * `[]` means this change is customer-facing and the message will be put into `HISTORY.rst`. `{}` means this change is not customer-facing and the message will **NOT** be included in `HISTORY.rst`.
     * If the component name is `Core`, the message will be written in `src/azure-cli-core/HISTORY.rst`. Otherwise, the message will be written in `src/azure-cli/HISTORY.rst`.
 2. [**Mandatory**] If it's a breaking change, the second part should be `BREAKING CHANGE` followed by a colon. In the case of hotfix, put `Hotfix` in this part. If it's related to fixing an issue, put `Fix #number` in this part. For other cases, this part could be empty.
-3. [**Recommendation**] If the change can be mapped into a command, then the next part could be the command name, followed by a colon.
-4. [**Recommendation**] Use the right verb with present-tense in original form to descibe what is done:
+3. [**Recommendation**] If the change can be mapped into a command, then the next part could be the command name starting with `az`, followed by a colon.
+4. [**Recommendation**] Use the right verb with present-tense in original form and capitalized first letter to descibe what is done:
```

Comment:
Maybe we can provide some examples.

Reply:
The examples are listed below. I made the key points bold now.